docs/_getting-started/index.md

```diff
@@ -442,33 +426,27 @@ curl -X POST http://127.0.0.1:8545 \
    --data '{"jsonrpc":"2.0", "method":"eth_accounts","params":[], "id":1}'
 ```
 
-This requires approval in Clef. Once approved, the following information is returned to the terminal:
+The following information is returned to the terminal:
 
 ```terminal
-
 {"jsonrpc":"2.0","id":1,"result":["0xca57f3b40b42fcce3c37b8d18adbca5260ca72ec"]}
-
 ```
 
 ### Sending Transactions
 
-Sending a transaction between accounts can also be achieved using Curl. Notice that the value of the transaction is a hexadecimal string in units of Wei. To transfer 0.1 ether, it is first necessary to convert this to Wei by multiplying by 10<sup>18</sup> then converting to hex. 0.1 ether is `"0x16345785d8a0000"` in hex. As before, update the `to` and `from` fields with the addresses in the Clef keystore.  
+It is possible to send transactions using raw curl requests too, but this requires unlocking the sender 
```

Comment:
@MariusVanDerWijden pinging you here to make sure you agree with this advice!

Reply:
Yeah makes sense